html/patterns/02-molecules-04-social-icons-social-icons/02-molecules-04-social-icons-social-icons.html

```diff
@@ -18,22 +18,13 @@
   <body class="">
   
 <ul class="social_media--list">
-      <li class="social_media--item">
-      <a href="http://www.twitter.com" title="Follow us on Twitter" class="social_media--icon Twitter">
-        Follow us on Twitter
-      </a>
-    </li>
-      <li class="social_media--item">
-      <a href="http://www.facebook.com" title="Like us on Facebook" class="social_media--icon Facebook">
-        Like us on Facebook
-      </a>
-    </li>
-      <li class="social_media--item">
-      <a href="http://www.linkedIn.com" title="Follow us on LinkedIn" class="social_media--icon LinkedIn">
-        Follow us on LinkedIn
-      </a>
-    </li>
-  </ul>
+      <div class="section-content">
```

Comment:
hmmm its weird to see this markup here. do you have a sense of why "section content" is showing up in social icons?